lib/mobile_application_platform/sign_up/configuration.rb

```diff
@@ -42,6 +42,13 @@ def provisioning_acceptable_status
         [200, 406, 412]
       end
 
+      def agreements_version_mapping
+        {
+          'v1' => 2,
+          'v2' => 3
+        }
+      end
```

Comment:
Would this agreement mapping make more sense to be included on the `TermsOfUseAgreement` model? I can see us needing this mapping in other process flows

Reply:
This one's a good point, I'm still a little on the fence with there to put it though. Basically we have our own internal versions (`v1`, `v2`), and the SignUpService has mappings that we only care about with regards to our interface with SignUpService. So I'm not actually sure if we will ever need this mapping outside of the `SignUp::Service` class... 

But on the other hand it's kinda weird to put information in the service config that's not relevant directly to the service (the `v1`, `v2` part). I suppose we've performed this straddling of concerns in the past by just making a `Constants` class, so maybe that's the right answer here... I'll let us refactor after this PR if that seems like a good solution